sb/robot/arenas/pirate_islands.py

```diff
@@ -0,0 +1,148 @@
+# Arena definition for 'Pirate Islands', the 2018 SourceBots game.
+import random
+from math import pi, cos, sin
+
+import math
+
+import pygame
+import pypybox2d
+
+from sb.robot.game_specific import TOKEN_ZONE_3, TOKEN_ZONE_2, TOKEN_ZONE_1, \
+    TOKEN_ZONE_0
+from sb.robot.arenas import Arena
+from sb.robot.arenas.arena import ARENA_MARKINGS_COLOR, ARENA_MARKINGS_WIDTH
+from sb.robot.markers import Token
+from ..game_object import GameObject
+
+#######
+
+PEDESTAL_DIAMETER_METERS = 0.3
+
+
+class Pedestal(GameObject):
+    @property
+    def location(self):
+        return self._body.position
+
+    @location.setter
+    def location(self, new_pos):
+        self._body.position = new_pos
+
+    @property
+    def heading(self):
+        return self._body.angle
+
+    @heading.setter
+    def heading(self, _new_heading):
+        self._body.angle = _new_heading
+
+    def __init__(self, arena):
+        self._body = arena._physics_world.create_body(position=(0, 0),
+                                                      angle=0,
+                                                      type=pypybox2d.body.Body.STATIC)
+
+        point_dist = PEDESTAL_DIAMETER_METERS / 2
+        self._body.create_polygon_fixture([(-point_dist, -point_dist),
+                                           (point_dist, -point_dist),
+                                           (point_dist, point_dist),
+                                           (-point_dist, point_dist)],
+                                          restitution=0.2,
+                                          friction=0.3)
+        super().__init__(arena)
+
+    @property
+    def surface_name(self):
```

Comment:
This would be so much nicer as simply `surface_name = 'sb/pedestal.png'`?

Reply:
fixed in dbdf7534ccea3a9cf59ebfaa3f5a3319c973ad0b